Library/Formula/libagar.rb

```diff
@@ -0,0 +1,37 @@
+class Libagar < Formula
+  desc "Cross-platform GUI toolkit"
+  homepage "http://libagar.org/"
+  url "http://stable.hypertriton.com/agar/agar-1.4.1.tar.gz"
+  sha256 "b0e62b754f134c3c0dd070a4fa62fa552654356eebab3d55e32d5d9b151a275e"
+  head "http://dev.csoft.net/agar/trunk", :using => :svn
+
+  depends_on "sdl"
+  depends_on "freetype"
+  depends_on "jpeg"
+  depends_on "libpng"
+
+  def install
+    ENV.deparallelize
+    system "./configure", "--prefix=#{prefix}"
+    system "make"
+    system "make", "install", "MANDIR=#{man}" # --mandir for configure didn't work
+  end
+
+  test do
+    (testpath/"test.c").write <<-EOS.undent
+      #include <agar/core.h>
+      #include <agar/gui.h>
+
+      int main() {
+        AG_Window *win;
+        if (AG_InitCore("test", AG_VERBOSE) == -1 || AG_InitGraphics(NULL) == -1) {
+          return 1;
+        } else {
+          return 0;
+        }
+      }
+    EOS
+    system ENV.cc, "test.c", "-o", "test", *`agar-config --cflags --libs`.split
```

Comment:
Rather than shelling out here, can we include the directories `agar-config --cflags --libs` flags? Presuming it occupies a standard layout, you should be fine with using `lib` and `include`.